packages/recomponents/src/components/r-popper/r-popper.story.js

```diff
@@ -14,6 +14,7 @@ storiesOf('Components.Popper', module)
             openOnMount: {default: boolean('Open on mount', false)},
             disabled: {default: boolean('Disabled', false)},
             offset: {default: number('Offset', 4)},
+            quantityOfOptions: {default: number('Quantity of options', 2)},
```

Comment:
I would personally prefer numberOfOptions but, of course, it's a matter of taste.